test/IBM.Watson.IntegrationTests/Icp4dIntegrationTests.cs

```diff
@@ -0,0 +1,44 @@
+﻿/**
+* Copyright 2019 IBM Corp. All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*
+*/
+
+using IBM.Cloud.SDK.Core.Authentication.Icp4d;
+using IBM.Watson.NaturalLanguageUnderstanding.v1;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace IBM.Watson.IntegrationTests
+{
+    //[TestClass]
+    public class Icp4dIntegrationTests

```

Comment:
Does this test actually work? I'd be a little wary of relying on instances we have now being stable enough to leave for integration tests.

Reply:
This test is skipped since I have `[TestClass]` commented out.